pkg/apis/serving/v1/revision_lifecycle.go

```diff
@@ -171,6 +171,13 @@ func (rs *RevisionStatus) PropagateAutoscalerStatus(ps *av1alpha1.PodAutoscalerS
 		return
 	}
 
+	if ps.IsScaleTargetInitialized() {
+		// Precondition for PA being initialized is SKS being active and
+		// that entices that |service.endpoints| > 0.
```

Comment:
nit - I know this is a copy/paste from below, but might as well fix: 

```suggestion
		// that implies that |service.endpoints| > 0.
```

(or maybe "requires" or "ensures", but definitely not "entices" :)).

Reply:
Copypasta but sure will fix.